File: app/src/main/java/com/mongst/vinlist2/Vin.java
package com.mongst.vinlist2;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity
public class Vin {

    Vin(String createVin, String notesVin) {
        this.createVin = createVin;
        this.notesVin = notesVin;
    }

    @PrimaryKey(autoGenerate = true)
    private int id;

    @ColumnInfo(name = "enter_vin")
    private String createVin;

    @ColumnInfo(name = "notes_vin")
    private String notesVin;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    String getCreateVin() {
        return createVin;
    }

//    public void setCreateVin(String createVin) {
//        this.createVin = createVin;
//    }

    String getNotesVin() {
        return notesVin;
    }

//    public void setNotesVin(String notesVin) {
//        this.notesVin = notesVin;
//    }
}

File: app/src/main/java/com/mongst/vinlist2/Vin.kt
package com.mongst.vinlist2

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "Vin")
data class Vin (
    @PrimaryKey(autoGenerate = true)
    val id: Int?,
    val createVin: String,
    val notesVin: String
)




//@Entity
//class Vin internal constructor(@field:ColumnInfo(name = "enter_vin")
//          internal val createVin: String, @field:ColumnInfo(name = "notes_vin")
//
//
//                               //    public void setCreateVin(String createVin) {
//                               //        this.createVin = createVin;
//                               //    }
//
//                               internal val notesVin: String) {
//
//    @PrimaryKey(autoGenerate = true)
//    var id: Int = 0
//
//    //    public void setNotesVin(String notesVin) {
//    //        this.notesVin = notesVin;
//    //    }
//}
